control-plane/config/eventing-kafka-broker/200-controller/200-controller-cluster-role.yaml

```diff
@@ -87,6 +87,12 @@ rules:
       - get
       - update
       - watch
+  - apiGroups:
+    - ""
+    resources:
+    - "secrets"
+    verbs:
+    - "*"
```

Comment:
```suggestion
      - get
      - list
      - create
      - update
      - delete
      - watch
```

Reply:
The secret permissions are already at line 81, can we add the missing verbs there instead?